src/git-duet/git-duet-cherry-pick/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"git-duet/internal/cmd"
+	"git-duet/internal/cmdrunner"
+)
+
+func main() {
+	err := cmdrunner.Execute(
+		cmd.New("cherry-pick"))
```

Comment:
I think you are right.  Assuming that they were probably using git-duet for the original commit, it should have both an author and a committer.

And, I don't think those should be touched during cherry-picking.  You are only moving the commit from one branch/ref to another, the "ownership" of the original commit isn't changing.

And in practice (at least in most workflows I know), this is usually only used to get individual commits onto a production release or hotfix branch (otherwise you'd be doing a merge or rebase of a whole branch to git all commits).  And, this often isn't even done by the original committer on large teams - it could be someone responsible for deploying the release or hotfix, not the original coder.

So, I think this is fine the way it is :)